### ISSUE TITLE
gurobi.py `get_objective` should return a LinExpr

### ISSUE THREAD

**Issue by brentian** [coin-or/python-mip#39]
now we have:

```python
def get_objective(self) -> LinExpr:
        return self.get_dbl_attr("ObjVal")
```

**Comment by tuliotoffolo** [comment]
Thanks for pointing this out. The issue will be fixed asap.

**Comment by h-g-s** [comment]
fixed in master, Ill release new version soon

**Comment by h-g-s** [comment]
fixed in 1.6.5, please try it.

**Comment by brentian** [comment]
> fixed in 1.6.5, please try it.

no problem now, thanks!
